classifiers/llm/deberta_review_classifier/code_snippet_refinery.md

```diff
@@ -0,0 +1,19 @@
+```python
+import requests
+import json
+
+API_KEY: str = "<API_KEY_GOES_HERE>"
+ATTRIBUTE: str = "headline" # only text attributes
+
+def deberta_review_classifier(record):
+    """
+    Uses toxic-bert via Hugging Face Inference API to classify toxicity in text.
+    """
+    api_token = API_KEY
```

Comment:
i dont think we need to reassign this variable to another variable and can use the API_KEY in the headers directly

Reply:
Correct! Changed it.